views/index.html

```diff
@@ -188,7 +187,12 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
   <!-- This needs an additional class of dark-bg when 'What our clients say' gets added -->
   <div class="copyright-block">
-    Copyright 2016 &copy; dwyl Ltd. All rights reserved. 104 Mary Datchelor Close, London SE5 7DY, UK
+    <p>
+      Copyright 2016 &copy; dwyl ltd, company number 09525434, registered in England &amp; based at 104 Mary Datchelor Close, London SE5 7DY, UK. All rights reserved.
```

Comment:
Thought the point was we have to list the Company & VAT number but don't *have* to publicise the address...?